backend/hitas/models/apartment.py

```diff
@@ -78,17 +78,35 @@ class Apartment(ExternalHitasModel):
 
     notes = models.TextField(blank=True, null=True)
 
-    # 'Hankinta-arvo' = Ensimmäisen kaupan (tai myyntihintaluettelon) kauppakirjahinta + yhtiölainaosuus
+    # 'Nykyinen Hankinta-arvo' = Joko ensimmäisen kaupan, tai sen puuttuessa myyntihintaluettelon hankinta-arvo
     @property
-    def first_sale_acquisition_price(self) -> Optional[Decimal]:
-        if self.first_sale_purchase_price is not None and self.first_sale_share_of_housing_company_loans is not None:
-            return self.first_sale_purchase_price + self.first_sale_share_of_housing_company_loans
+    def current_acquisition_price(self) -> Optional[Decimal]:
+        first_sale_price = self.first_sale_acquisition_price
+        if first_sale_price:
+            return first_sale_price
+
+        catalog_price = self.catalog_acquisition_price
+        if catalog_price:
+            return catalog_price
 
+        return None
```

Comment:
This property is not used anywhere anymore (except a test). Reading the ticket's description this property seems not needed anymore, so this can be removed
To separate the catalogue/first sale values we can just do a little bit of logic in the frontend, since we already have both values there.

Reply:
Done